### DISCUSSION TITLE
Min/Max functions

### DISCUSSION THREAD

**Discussion by Dietr1ch** [casey/just#2453] [Q&A] [answered]
I'm in need for something like `min(num_cpus(), K)` to avoid underlying issues with some tools.

Is there a good way of getting that? I think I'm a bit held back by #11 to use the shell, but maybe there's a better workaround.

**Comment by laniakea64** [comment, marked as answer]
> I think I'm a bit held back by #11 to use the shell, 

How so?  Why not using the [`shell()` function](https://just.systems/man/en/functions.html#external-commands)?

**Comment by Dietr1ch** [comment]
I completely missed that function when reading the docs, thanks!
